python/lsst/ts/xml/data/sal_interfaces/LEDProjector/LEDProjector_Telemetry.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALTelemetrySet.xsl"?>
+<SALTelemetrySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALTelemetrySet.xsd">
+  <SALTelemetry>
+    <Subsystem>LEDProjector</Subsystem>
+    <EFDB_Topic>LEDProjector_LEDStates</EFDB_Topic>
```

Comment:
Usually telemetry topics are something you are constantly monitor and publishing. Things like temperature etc. A state is probably something that will change asynchronously so, there's no need to constantly publish it.